error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/ReactorRules.java

```diff
@@ -1232,6 +1232,24 @@ StepVerifier.Step<T> after(StepVerifier.Step<T> step, T object) {
     }
   }
 
+  /**
+   * Prefer {@link StepVerifier.LastStep#verifyComplete()} over {@code
+   * StepVerifier.Step#expectNextCount(0).verifyComplete()} or its alternatives.
+   */
+  static final class StepVerifierStepExpectNextCountZeroVerifyComplete<T> {
+    @BeforeTemplate
+    Duration before(StepVerifier.Step<T> step) {
+      return Refaster.anyOf(
+          step.expectNextCount(0L).expectComplete().verify(),
```

Comment:
Do we need to check for `...expectComplete().verify()` as it is already checked in `StepVerifierLastStepVerifyComplete` class ?

Reply:
Build error gave the same [answer](https://github.com/PicnicSupermarket/error-prone-support/actions/runs/4489690922/jobs/7895839277#step:6:2231) :trollface: